# motion
verbose_motion: true
use_ground_truth: false
variance: true
qa: 1
qx: 1
qy: 1
qz: 1
enable_gps: true
r_gps_x: 5 #1 
r_gps_y: 5 #1
r_gps_z: 5 #1
enable_ma#gnet: true
r_mgn_a: 1.0e-1
enable_baro: false
r_bar_z: 1.0e-1
enable_sonar: true
r_snr_z: 1.0e-1
motion_iter_rate: 20
